sdk/keyvault/azure-security-keyvault-common/README.md

```diff
@@ -0,0 +1,5 @@
+# Azure Key Vault Common Package client library for C++
+
+### License
+
+Azure SDK for C++ is licensed under the [MIT](https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/core/azure-core/LICENSE) license.
```

Comment:
Should libraries have their own license file each, or reference License.txt at the root of the repo? I think, it is second.